ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageSingleVulnerabilities.tsx

```diff
@@ -184,11 +189,8 @@ function ImageSingleVulnerabilities({ imageId }: ImageSingleVulnerabilitiesProps
                     >
                         <PageSection variant="light" component="div" isFilled>
                             <WorkloadTableToolbar
-                                // TODO: wire up the actual default filters in this component
-                                defaultFilters={{
-                                    Severity: ['Critical'],
-                                    Fixable: ['Fixable'],
-                                }}
+                                // Default filters do not apply to singles pages
+                                defaultFilters={emptyDefaultFilters}
```

Comment:
Rather than requiring each caller to pass empty filters, should we bake the empty defaults into the WorkloadTableToolbar?